spec/models/validations/sales/soft_validations_spec.rb

```diff
@@ -912,6 +912,15 @@
         expect(record).not_to be_discounted_ownership_value_invalid
       end
     end
+
+    context "when it is a 2024 log" do
```

Comment:
Do we want to adjust the soft validation to not run for 2024? I guess it doesn't really matter because it'll be caught by the hard one before anything from the soft comes into play, but it seems weird to still have it.

Reply:
I think it's already only running for 2023! Added this test because it wasn't covered